pkg/utils/ingress.go

```diff
@@ -30,7 +30,8 @@ func SetIngressClassEnabled(kc kubernetes.Interface) {
 
 	var isPresent bool
 	timeout := int64(120)
-	_, ingClassError := kc.NetworkingV1beta1().IngressClasses().List(context.TODO(), metav1.ListOptions{TimeoutSeconds: &timeout})
+	// should only work for k8s 1.19+ clusters
+	_, ingClassError := kc.NetworkingV1().IngressClasses().List(context.TODO(), metav1.ListOptions{TimeoutSeconds: &timeout})
```

Comment:
There's no problem if you run this Kubernetes 1.17?

Reply:
in any k8s version prior to 1.19, it should throw an error, and `ingressClassEnabled` should get set to false.